base-system/tools/installer/install.sh

```diff
@@ -160,6 +185,12 @@ cp --verbose -rf "$ISO_DIR/EFI/" "$SYSTEM_MNT"
 cp --verbose -rf "$ISO_DIR/checksums" "$SYSTEM_MNT"
 cp --verbose -rf "$SERVER_CONFIG" "$SYSTEM_MNT/huronOS/data/configs/sync-server.conf"
 
+# If a password was passed, update the password
+if [ -n "$NEW_PASSWORD" ]; then
```

Comment:
I thought it would be simpler to leave it optional, but I aknowledge my security pov is not the best, so if you suggest that is better to make the password a required field I'll change it.

Reply:
> I thought it would be simpler to leave it optional, but I aknowledge my security pov is not the best, so if you suggest that is better to make the password a required field I'll change it.

Let's leave it optional for now, after we're able to test this script on several scenarios we should move this to required.